.travis.yml

```diff
@@ -11,4 +11,16 @@ cache:
     - node_modules
 
 after_success:
-  cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
\ No newline at end of file
+  cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+
+jobs:
+  include:
+    - stage: release
+      node_js: lts/*
+      script:
+        - echo "skipping tests"
```

Comment:
I think this repo has tests? We should probably run them before cutting a release?

Reply:
The semantic-release recommends skipping tests in the release stage condition if the tests were run before in the previous stage.

Reply:
The previous stage being the Pull Request commit status publisher?

Reply:
It can be any previous stage, the documentation doesn't say about a specific stage. Probably like Circle CI, the test stage is run as one of the first ones.